### DISCUSSION TITLE
KitchenOwl Import

### DISCUSSION THREAD

**Discussion by DeanFoley** [mealie-recipes/mealie#2884] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

KitchenOwl is another similar app to Mealie, being a self-hostable server app for organising recipes and shopping lists. Migrating from KitchenOwl to Mealie would be much easier with an import function similar to the others on offer!

### Please provide a concise description of the feature that would resolve your issue.

Ability to import data from KitchenOwl servers to a Mealie server.

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Do you have an example of what an export from KitchenOwl looks like?

**Reply by DeanFoley** [reply]
Got an export, let me just cut it down to not be so big and I'll upload it here 👍

**Reply by DeanFoley** [reply]
[Here you go!](https://gist.github.com/DeanFoley/5fb4b225e41908cfde4c1965febcc6e8)

**Reply by michael-genson** [reply]
Thanks! This looks reasonably importable. Is there any other data besides the JSON? It looks like there are references to images, but we'd obviously need the image files

**Reply by DeanFoley** [reply]
There are images, but the export doesn't provide the option to also export those (or any options at all, actually...)

Sounds like a nice addition on the KitchenOwl side! But for now it would be a pretty manual operation saving all your images from your existing set-up.

Otherwise, nope, that's it, just a big ol' JSON payload.

**Comment by DeanFoley** [comment]
Oh I forgot to link what the app is! [KitchenOwl](https://github.com/TomBursch/kitchenowl)
